src/libsyntax/feature_gate.rs

```diff
@@ -570,6 +570,9 @@ declare_features! (
     // #[repr(transparent)] on unions.
     (active, transparent_unions, "1.37.0", Some(60405), None),
 
+    // `[x; N]` where `x` is a constant (RFC 2203).
```

Comment:
```suggestion
    // Allows `[x; N]` where `x` is a constant (RFC 2203).
```

Can you also drive-by fix the two feature gates above that don't conform to the style?